src/store/model/bestbuy.ts

```diff
@@ -227,6 +227,13 @@ export const BestBuy: Store = {
 			model: 'xlr8 rgb',
 			series: '3090',
 			url: 'https://www.bestbuy.com/site/pny-geforce-rtx-3090-24gb-xlr8-gaming-epic-x-rgb-triple-fan-graphics-card/6432657.p?skuId=6432657&intl=nosplash'
+		},
+		{
+			brand: 'amd',
+			cartUrl: 'https://api.bestbuy.com/click/-/6438942/cart',
```

Comment:
I noticed the item number in the API call and the url are for the 3900 instead of the 3950.

Reply:
Good catch!  I also went ahead and added all the new ryzen cpu's ;)

Reply:
Awesome, thanks!